packages/fakta-overstyr-beregning/src/components/OverstyrBeregningFaktaForm.tsx

```diff
@@ -0,0 +1,188 @@
+            }, 5000);
+            return () => clearTimeout(delayTimer);
+        }
+        return undefined;
+    }, [submitDelay]);
+
+    const aktivitetSchema = Yup.object().shape({
+        arbiedsgiverAktørId: Yup.string(),
+        arbeidsgiverOrgnr: Yup.string(),
+        inntektPrAar: Yup.number()
+            .typeError(intl.formatMessage({ id: 'OverstyrInputForm.InntektFeltTypeFeil' }))
+            .required(intl.formatMessage({ id: 'OverstyrInputForm.InntektFeltPakrevdFeil' })),
+        refusjonPrAar: Yup.number()
+            .typeError(intl.formatMessage({ id: 'OverstyrInputForm.RefusjonFeltTypeFeil' }))
+            .required(intl.formatMessage({ id: 'OverstyrInputForm.RefusjonFeltPakrevdFeil' })),
+    });
+
+    const periodeSchema = Yup.object().shape({
+        skjaeringstudspunkt: Yup.string(),
+        aktivitetliste: Yup.array().of(aktivitetSchema)
+    });
+
+    const validationSchema = Yup.object().shape({
+        kode: Yup.string().required(),
+        begrunnelse: Yup.string().required(intl.formatMessage({ id: 'OverstyrInputForm.BegrunnelseErPåkrevd' })),
+        perioder: Yup.array().of(periodeSchema),
+    });
+
+    /**
+     * @param firmaIdent Kan være enten orgnr eller aktørId
+     * @returns utledet firmanavn fra arbeidsgiverOpplysningerPerId
+     */
+    const utledFirmaNavn = (firmaIdent: string) => {
+        const firma = Object.values(arbeidsgiverOpplysningerPerId).find((arbeidsgiver => arbeidsgiver.identifikator === firmaIdent));
+        return (firma) ? firma.navn : intl.formatMessage({ id: 'OverstyrInputForm.UkjentFirma' });
+    }
+
+    const utledBegrunnelse = () => aksjonspunkter.find((ap) => ap.definisjon.kode === aksjonspunktCodes.OVERSTYR_BEREGNING_INPUT).begrunnelse || ''
+
+    /**
+     * Formik liker ikke null i value feltene, null verdier kan forekomme fra backend. 
+     * "Oversetter" null verdier i skjemafeltene til undefined
+     */
+    const initialValues: OverstyrInputForBeregningDto = {
+        kode: aksjonspunktCodes.OVERSTYR_BEREGNING_INPUT,
+        begrunnelse: utledBegrunnelse(),
+        perioder: [{
+            ...overstyrInputBeregning, aktivitetliste: overstyrInputBeregning.aktivitetliste.map((aktivitet) => ({
+                ...aktivitet,
+                "inntektPrAar": aktivitet.inntektPrAar || undefined,
+                "refusjonPrAar": aktivitet.refusjonPrAar || undefined,
+            }))
+        }],
+    }
+
+    const erDeaktivert = (isValid) => readOnly || !submittable || !isValid || !harEndredeVerdier || submitDelay;
+
+    return (
+        <div className={styles.container}>
+            <VerticalSpacer thirtyTwoPx />
+            <Formik
+                initialValues={initialValues}
+                onSubmit={values => {
+                    setSubmitDelay(true);
+                    submitCallback([values]);
+                    // eslint-disable-next-line no-console
+                    console.log(JSON.stringify(values, null, 2));
```

Comment:
Fjern console

Reply:
Jeg har latt denne ligge igjen for testingens skyld, men kan heller legge den til igjen om det trengs